news/2025/2025-04-08-beatmap-spotlights-revival-spring-2025.md

```diff
@@ -0,0 +1,269 @@
+
+Without a doubt, this boss chart deserves its place as the top spotlight pick, and we are confident that it will bring you along for the best of experiences and most rewarding of rides!
+
+#### Playlist A
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/BISOAEcPHaU" frameborder="0" allowfullscreen></iframe>
+</div>
+
+- [25-ji, Nightcord de. x KAITO - Samsa (\_Yiiiii) \[\[7K\] Dzar03's Hard\]](https://osu.ppy.sh/beatmapsets/2154243#mania/4578895), suggested by [Maxus](https://osu.ppy.sh/users/4335785)
+- [Kendrick Lamar - United In Grief (Disguise) \[\[4K\] elexire's Insane\]](https://osu.ppy.sh/beatmapsets/2210015#mania/4690649), suggested by [Crumpey](https://osu.ppy.sh/users/3518705)
+- [Lime - Campanella (Tailsdk) \[NoName's 7K Insane\]](https://osu.ppy.sh/beatmapsets/2146184#mania/4520611), suggested by [Maxus](https://osu.ppy.sh/users/4335785)
+- [xi - Rebellion Trigger (FerdiXIA-) \[\[4K\] Ichi's Insane\]](https://osu.ppy.sh/beatmapsets/1855635#mania/4063550), suggested by [Ilham](https://osu.ppy.sh/users/3057154)
+- [A.SAKA - Nanatsu Issenzakura (YyottaCat) \[\[5K\] Lencten Blostma \<Spring Blossoms\>\]](https://osu.ppy.sh/beatmapsets/2194881#mania/4643973), suggested by [Antalf](https://osu.ppy.sh/users/8793773)
+- [siromaru - Imperial Prime Aggression (AHHHHHHHHHHHHHH) \[\[4K\] Extra\]](https://osu.ppy.sh/beatmapsets/2089498#mania/4571836), suggested by [Carpihat](https://osu.ppy.sh/users/10085090)
+- [X JAPAN - Kurenai (Alptraum) \[\[4K\] Crying in deep red\]](https://osu.ppy.sh/beatmapsets/2101873#mania/4409471), suggested by [Carpihat](https://osu.ppy.sh/users/10085090)
+- [Camellia - We Could Get More Machinegun Psystyle! (And More Genre Switches) (Rio1-) \[\[6K\] Relentless\]](https://osu.ppy.sh/beatmapsets/2211806#mania/4685601), suggested by [Antalf](https://osu.ppy.sh/users/8793773)
+- [Ludicin - Everlasting Eternity (Critical\_Star) \[\[7K\] Timeless\]](https://osu.ppy.sh/beatmapsets/2099753#mania/4404687), suggested by [Maxus](https://osu.ppy.sh/users/4335785)
+
+Playlist B, C and D will be available on the [Beatmap Spotlights article](/wiki/Beatmap_Spotlights) as we open the playlist lobbies, so stay tuned!
+
+## Beatmap Spotlight Curator applications
+
+None of this would be possible without our [curators](https://osu.ppy.sh/groups/48)' hard work and efforts.
+
+That being said, we are looking towards expanding the team further in order to make picking the beatmaps more deliberate and effective.
```

Comment:
```suggestion
That being said, we are looking to expand the team in order to streamline the beatmap selection process.
```
What does adding more people have to do with how deliberate your picking strategy is?